### PR TITLE
move commands from brave-browser -> brave-core

### PR DESCRIPTION
**Pull request by bridiver**
Resolves https://github.com/brave/brave-browser/issues/10973
Requires https://github.com/brave/brave-browser/pull/10885
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,29 @@
...
+    "start": "node ./build/commands/scripts/commands.js start",
+    "network-audit": "node ./build/commands/scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test --disable-doh",
+    "push_l10n": "node ./build/commands/scripts/commands.js push_l10n",
+    "pull_l10n": "node ./build/commands/scripts/commands.js pull_l10n",
+    "chromium_rebase_l10n": "node ./build/commands/scripts/commands.js chromium_rebase_l10n",
+    "lint": "node ./build/commands/scripts/commands.js lint",
+    "test": "node ./build/commands/scripts/commands.js test",
+    "test:scripts": "jest build/commands/lib build/commands/scripts",
+    "test-security": "npm run audit_deps && node ./build/commands/scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test",
+    "tslint": "tslint --project tsconfig-lint.json \"components/**/*.{ts,tsx}\"",
```
Comment (antonok-edm) [inline package.json:29]: Looks like this change by necessity allows C++ lints (`npm run lint`) and typescript lints (`npm run tslint`) to be run independently from the same directory, which is awesome.

It does seem a little weird to me to have a specifically-named command `tslint` and generically-named command `lint` with similar behavior for different languages; I'd recommend something like the following:
- `npm run tslint` as specified above
- `npm run cpplint` to take the current behavior of `npm run lint`
- Optionally, also `npm run lint` as a convenient way to run all of the above together

This scheme would be pretty natural to extend to other languages too, e.g. for our Rust FFIs.

 I'd also like to see these aliased the same way in `brave-browser` (currently `tslint` is missing from https://github.com/brave/brave-browser/pull/10885).

Comment (bridiver) [reply package.json:29]: it's just for backwards compat to get this through CI and actually I want to get rid of all the aliases commands in brave-browser which are also there for backwards compat. This is just the first step in moving these files

**Comment by petemill** [comment]
This is how I moved the commits from brave-browser to brave-core for only these files...

Command that was run on a temporary brave-browser repo clone:
```
git filter-repo --path scripts/ --path lib/ --to-subdirectory-filter build/commands
```

Then that was pulled in to brave-core as a branch with `git pull ... --allow-unrelated-histories`. Then @bridiver 's changes were rebased on top of that.

**Comment by bridiver** [comment]
temporarily added ci skip labels to re-run windows

**Comment by bridiver** [comment]
travis failed due to `npm run lint` -> `npm run tslint` - @mihaiplesa will remove it from travis and add to jenkins

**Pull request merged**